zetaclient/context/app.go

```diff
@@ -70,6 +70,10 @@ func (a *AppContext) Config() config.Config {
 	return a.config
 }
 
+func (a *AppContext) Logger() zerolog.Logger {
```

Comment:
Let's not expose context's logger to the outer scope.

Reply:
is there an issue you see with that , 
I am using it here https://github.com/zeta-chain/zeta-node/blob/2cce32c2fefc3ef7be37c797a47d3a6cd9a51905/zetaclient/zetacore/client_tss_migration_listners.go#L55-L55